h2o-r/tests/runitUtils/utilsR.R

```diff
@@ -376,6 +376,36 @@ function(testDesc, test) {
     PASS()
 }
 
+doSuite<-
+function(suiteDesc, suite, run_in_isolation=TRUE) {
+    suiteTest <- function() {
+        warnings <- c()
+        errors <- c()
+        lapply(suite$tests, function(test_name) {
+          cat("\n")
+          cat("Running", test_name, "\n")
+          if(run_in_isolation) h2o.removeAll()
+          tryCatch(
+              test_that(test_name, withWarnings(do.call(test_name, list(), envir=suite$envir))),
+              warning = function(w) warnings <<- c(warnings, w),
+              error = function(e) errors <<- c(errors, e$message)
+          )  
+          # do.call(test_name, list(), envir=envir)
+        })
+        if(length(warnings) > 0)
+            warning(paste("\n", warnings, "\n"))
+        if(length(errors) > 0)
+            stop(paste("Failing tests:\n", errors, "\n"), call. = FALSE)
+    }
+    doTest(suiteDesc, suiteTest)
+}
+
```

Comment:
same as for Python, added this small runner to run R tests functions in isolation and see clearly which test fails if any.